packages/utils/src/lib/throttle.ts

```diff
@@ -117,3 +226,41 @@ export function throttleToNextFrame(fn: () => void): () => void {
 		}
 	}
 }
+
+/**
+ * Gets the current target FPS (set to device's natural maximum FPS after detection).
+ * @returns the current target FPS (between 30-120)
+ * @internal
+ */
+export function getCurrentFps(): number {
+	return targetFps
+}
+
+/**
+ * Resets the FPS detection system back to default settings.
+ * Useful for testing or when you want to re-detect the device's natural FPS.
+ * @internal
+ */
+export function resetAdaptiveFps(): void {
```

Comment:
Do we still need `resetAdaptiveFps` and `setTargetFps`, it looks like they are not being used? Maybe it's they are useful for testing?

Reply:
Happy to keep it then👍